js/actions/windowActions.js

```diff
@@ -243,20 +243,6 @@ const windowActions = {
   },
 
   /**
-   * Dispatches a message to the store to set a preview frame.
-   * This should only be called internally by `WINDOW_SET_TAB_HOVER_STATE`
-   * when we need to delay updating the preview frame value
-   *
-   * @param {Object} frameKey - the frame key for the webview in question.
-   */
-  setPreviewFrame: function (frameKey) {
```

Comment:
we don't use it anymore. `frameStateUtilsetPreviewFrameKey(..args)` now checks for preview mode and returns if there is none. I guess the only use case for that was to delay tab previews for one use case, which we don't have anymore